### ISSUE TITLE
Deletion in replace mode deletes actual character

### ISSUE THREAD

**Issue by hacksoi** [jaredpar/VsVim#2380]
In gvim, when I press backspace in replace mode, it basically just replaces the character under the cursor with a space and moves the cursor back one.

In vsvim, however, it deletes the character under the cursor.

I much prefer gvim's approach.

Thank you for your work.

**Comment by ricksladkey** [comment]
This issue is a duplicate of issue #653 and was fixed in PR #2157. VsVim and vim now work the same way in replace mode!

**Comment by hacksoi** [comment]
Hi ricksladkey,

When will the next VsVim update be released?
The link in "Latest Builds" in the readme doesn't seem to work.

**Comment by ricksladkey** [comment]
@hacksoi I will fix the link! Since you are now on the new version, please verify the replace bug is fixed and keep an eye out for any unusual behavior. Any and all feedback welcome.

@jaredpar is almost ready to make a release. I'm not sure when that will be, but if I had to make a guess, it may be a few weeks because various things have to happen first.

**Comment by jaredpar** [comment]
>  it may be a few weeks because various things have to happen first.

That seems about right. ~1 week to get the last of the issues fixed + couple of days of dogfooding

**Comment by hacksoi** [comment]
the bug is indeed fixed, along with a lot of other bugs!
